README.md

```diff
@@ -290,6 +301,9 @@ environment variables take precedence over configuration pulled from a Kubernete
 | GITHUB_TOKEN   | Github token               | Required with `AUTH_TYPE` of `github` |
 | ROLE_ID        | Vault AppRole Role_ID      | Required with `AUTH_TYPE` of `approle` |
 | SECRET_ID      | Vault AppRole Secret_ID    | Required with `AUTH_TYPE` of `approle` |
+| K8S_MOUNT_POINT | Kuberentes Auth Mount Point | Required with `AUTH_TYPE` of `k8s` |
+| K8S_ROLE       | Kuberentes Auth Role      | Required with `AUTH_TYPE` of `k8s` |
+| K8S_TOKEN_PATH | Path to JWT for Kubernetes Auth  | Optional for `AUTH_TYPE` of `k8s` defaults to `/var/run/secrets/kubernetes.io/serviceaccount/token` |
```

Comment:
I believe for this to be available at all, you have to override some property in the Argo manifests that says `autoMountServiceAccountToken: false` or something. This is also the requirement for the `generate -s <secret-name> .` invocation to work. So I think we should make a clear note about that, especially since out of the box, this wouldn't work (I think).